src/drivers/gnss/septentrio/septentrio.h

```diff
@@ -0,0 +1,413 @@
+	uint8_t                                        _jamming_state{0};                                                                ///< Receiver jamming state
+
+	// uORB topics and subscriptions
+	gps_dump_s                                     *_dump_to_device{nullptr};                                                        ///< uORB GPS dump data (to the receiver)
+	gps_dump_s                                     *_dump_from_device{nullptr};                                                      ///< uORB GPS dump data (from the receiver)
+	sensor_gps_s                                   _report_gps_pos{};                                                                ///< uORB topic for gps position
+	satellite_info_s                               *_p_report_sat_info{nullptr};                                                     ///< Pointer to uORB topic for satellite info
+	GPSSatelliteInfo                               *_sat_info{nullptr};                                                              ///< Instance of GPS sat info data object
+	uORB::Publication<gps_dump_s>                  _dump_communication_pub{ORB_ID(gps_dump)};                                        ///< uORB topic for dump GPS data
+	uORB::Publication<gps_inject_data_s>           _gps_inject_data_pub{ORB_ID(gps_inject_data)};                                    ///< uORB topic for injected data to the receiver
+	uORB::PublicationMulti<sensor_gps_s>           _report_gps_pos_pub{ORB_ID(sensor_gps)};                                          ///< uORB topic for gps position
+	uORB::PublicationMulti<satellite_info_s>       _report_sat_info_pub{ORB_ID(satellite_info)};                                     ///< uORB topic for satellite info
+	uORB::SubscriptionMultiArray<gps_inject_data_s, gps_inject_data_s::MAX_INSTANCES> _orb_inject_data_sub{ORB_ID::gps_inject_data}; ///< uORB topic about data to inject to the receiver
+
+	// Reading and update rates
+	// NOTE: Both `_rate_vel` and `_rate_lat_lon` seem to be used in exactly the same way.
+	uint64_t _interval_rate_start{0};            ///< Start moment of measurement interval for position and velocity messages from the receiver
+	float    _rate{0.0f};                        ///< uORB position data publish rate (Hz)
+	uint8_t  _rate_count_vel;                    ///< Velocity messages from receiver in current measurement interval
+	uint8_t  _rate_count_lat_lon{};              ///< Position messages from receiver in current measurement interval
+	unsigned _last_rate_rtcm_injection_count{0}; ///< Counter for number of RTCM messages
+	float    _rate_vel{0.0f};                    ///< Velocity message rate from receiver in the last measurement interval (Hz)
+	float    _rate_lat_lon{0.0f};                ///< Position message rate from receiver in the last measurement interval (Hz)
```

Comment:
These two values are always set in the exact same locations and seem to always contain the same value.